src/quo/components/list_items/network_list/view.cljs

```diff
@@ -41,14 +41,14 @@
    [:catn
     [:props
      [:map {:closed true}
-      [:network-name {:optional true} [:maybe keyword?]]
-      [:fiat-value {:optional true} [:maybe :string]]
-      [:label {:optional true} [:maybe :string]]
-      [:token-value {:optional true} [:maybe [:string]]]
+      [:network-name [:maybe keyword?]]
+      [:fiat-value [:maybe :string]]
+      [:label [:maybe :string]]
+      [:token-value [:maybe [:string]]]
```

Comment:
Here i meant it specifically for the `:token-value` prop schema. Instead of `[:token-value {:optional true} [:maybe [:string]]]`, it should just be `[:token-value {:optional true} [:maybe :string]]`. The nesting around `:string` is not necessary and leads to a schema error.

These changes brought another question tho. Are all the props optional here? In case some of them are mandatory, you can remove the `{:optional true}` and `[:maybe ...]` e.g. if `fiat-value` is mandatory, the schema would just be `[:fiat-value :string]`

Reply:
schema has been updated